### DISCUSSION TITLE
نمایش دامنه و ساب دامین و مخفی کردن آی یرور اصلی

### DISCUSSION THREAD

**Discussion by miladsf** [hiddify/Hiddify-Manager#182] [Q&A سوالات رایج]
سلام وقت بخیر 
اول تشکر می کنم از تمام زحماتی که میکشین .
سوالم این بود که با این پنل میشه دامنه و سابو مخفی کرد چون عملا قابل دیدنه و راحت میشه دامینو ف*لتر کرد.
و اینکه راهی برای مخفی کردن آی پی سرور وجود داره چون با سایت های تشخیص آی پی راحت میشه آی پی سرور اصلی رو بدست آورد.

**Comment by just1random** [comment]
در صورت استفاده از Domain Fronting و به شرط استفاده از CDN خارجی، هم دامنه و هم آی پی سرور مخفی خواهد موند.
در صورت استفاده از CDN و به شرط استفاده از CDN خارجی، آی پی سرور مخفی خواهد موند.
در سایر موارد آی پی سرور قابل دسترسی است.

**Reply by digitalist58** [reply]
با سلام
روی پنل هایدیفای چطور میشه از Domain Fronting استفاده کرد؟
اگر هم براتون مقدوره کلا یه توضیحی بدید که Domain Fronting رو چطور میشه فعال کرد.
با تشکر
